keychain_linux.cpp

```diff
@@ -30,10 +30,12 @@
 
 namespace keychain {
 
+namespace {
```

Comment:
anonymous namespaces removes the members from the public ABI of the compilation unit